tss-esapi/src/structures/pcr/selection.rs

```diff
@@ -27,18 +29,50 @@ impl PcrSelection {
         PcrSelection {
             hashing_algorithm,
             size_of_select,
-            selected_pcrs: selected_pcr_slots.iter().cloned().collect(),
+            selected_pcrs: Self::to_internal_representation(selected_pcr_slots),
         }
     }
 
-    pub fn hashing_algorithm(&self) -> HashingAlgorithm {
+    /// Returns the hashing algorithm for the selection
+    pub const fn hashing_algorithm(&self) -> HashingAlgorithm {
         self.hashing_algorithm
     }
 
-    pub fn selected_pcrs(&self) -> &BitFlags<PcrSlot> {
-        &self.selected_pcrs
+    /// Returns 'Size of Select'
+    ///
+    /// OBS! This is not the same as how many [PcrSlot]
```

Comment:
What does `OBS` mean?

Reply:
Sorry that is swedish. It should be NB or Note.

Reply:
:sweden:

Reply:
Fixed